packages/circuit-ui/components/Toggle/Toggle.tsx

```diff
@@ -44,25 +44,29 @@ export interface ToggleProps extends SwitchProps {
 const textWrapperStyles = ({ theme }: StyleProps) => css`
   display: block;
   margin-left: ${theme.spacings.kilo};
-  cursor: pointer;
 
   ${theme.mq.untilKilo} {
     margin-left: 0;
     margin-right: ${theme.spacings.kilo};
   }
+`;
+
+const ToggleTextWrapper = styled('div')(textWrapperStyles);
+
+const labelStyles = () => css`
+  cursor: pointer;
 
   .${CLASS_DISABLED} & {
     color: var(--cui-fg-normal-disabled);
   }
 `;
 
-const ToggleTextWrapper = styled('label')(textWrapperStyles);
-
-const explanationStyles = css`
-  color: var(--cui-fg-subtle);
+// This component is rendered as a `label` element below.
+const ToggleLabel = styled(Body)<{ htmlFor: string }>(labelStyles);
```

Comment:
Is this to make sure we get `Body` styles? Is this also a small bug in the previous implementation?

Reply:
The `label` was also wrapped in the `Body` component before, which was in turn wrapped inside a `label` element alongside the `explanation`. 

The wrapper element is now a simple `div` and the `label` is still wrapped in the `Body` component, now rendered as a `label` element.

Reply:
Done in 5758d72035b30611fb6a6510599881ed2e156194.

Reply:
Ah right! I saw the `<p>`s in the snapshots but didn't realize they were for styling. Thank you!